builds/development/css/audience.css

```diff
@@ -3268,129 +3287,129 @@ u {
 }
 
 @media (max-width: 767px) {
-  .top .logo a {
-    margin: 0 auto;
-    width: 50%;
+  .top .logo a img {
+    width: 100%;
   }
 }
 
-.top .department {
-  color: #fff;
-  font-family: "Helvetica Neue", "Helvetica", "Arial", sans-serif;
-  letter-spacing: 1px;
+.top .logo .svgLinkContainer {
+  display: block;
+  position: relative;
+  height: 1px;
+  width: 100%;
+  padding-bottom: 29.95839%;
+  z-index: 2;
 }
 
-.top .department > a {
-  display: inline-block;
-  padding: 2px 10px;
-  line-height: 1.2em;
+.top .logo .svgLinkContainer::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
 }
 
-@media (min-width: 768px) {
-  .top .department > a {
-    border-left: solid 1px #c28e0e;
-  }
+.top .logo .svgContainer {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: -1;
+  padding-right: 0px;
 }
 
 @media (max-width: 767px) {
-  .top .department > a {
-    border: none;
+  .top .logo .svgLinkContainer {
+    margin: 0 auto;
+    width: 50%;
+    padding-bottom: 14.9792%;
   }
 }
 
-.top .department > a:link {
-  color: inherit;
-  font-weight: 100;
-  text-decoration: none;
+.top .department {
+  color: #fff;
+  font-family: "Archivo Narrow", "Arial Narrow", "Arial", sans-serif;
+  letter-spacing: 0;
+  padding-left: 15px;
+  padding-right: 15px;
 }
 
-.top .department > a:visited {
-  color: inherit;
-  font-weight: 100;
-  text-decoration: none;
+.top .department > a {
+  display: inline-block;
+  line-height: 1.1em;
+  border: none;
+  padding: 0;
```

Comment:
Another focus style for keyboard navigation:
```
&:focus{
    outline:2px dotted #fff;
    outline-offset:1px;
}
```
![site name on focus example](https://user-images.githubusercontent.com/6598084/36607337-8e85ebca-1883-11e8-85fd-7ad8b9bb9ef8.png)

@jengarza @bholaday Thoughts?